spec/site_prism/page_objects/pages/cases/data_request_page.rb

```diff
@@ -1,7 +1,7 @@
 module PageObjects
   module Pages
     module Cases
-      class DataRequestPage < SitePrism::Page
+      class DataRequestPage < PageObjects::Pages::Base
         set_url '/cases/{case_id}/data_requests{/new}'
```

Comment:
What is `{/new}` referring to here - this is not a syntax I am familiar with. Usually interpolated values come after the `/`

Reply:
This was already in place - I'm not sure exactly why they did it this way 🤔 

I'll read up about it sometime!